content/800-data-platform/200-projects/100-add-project/130-import-existing-project-new-db.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Import an existing project with a new database'
+metaTitle: 'Import an existing JavaScript/TypeScript project with a new database in the Prisma Data Platform'
+metaDescription: 'If you want to test how your Node.js application is created, managed, and runs in the Prisma Data Platform, you can import your repository and connect the application to a new database. The Prisma Data Platform applies all schema migrations to the new database.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Import your application and use an empty database to explore and test project creation and the features in the Prisma Data Platform.
+
+You can use the steps provided in [Import an existing project with an existing database](/data-platform/projects/add-project/import-existing-project-existing-db) to import a project.
+
+At step 4.1, make sure that you provide a connection string to an empty database.
+
+Project creation uses the Prisma schema from the GitHub repository to migrate the current schema.
```

Comment:
We actually do not migrate empty databases. So the copy at least needs to be changed here. Tho this makes me think maybe we dont need this page at all? As the flow is exactly the same as with "existing database". WDYT?